docs/sources/flow/reference/components/prometheus.exporter.redis.md

```diff
@@ -41,6 +41,7 @@ Omitted fields take their default values.
 | `check_single_keys`           | `list(string)` | List of single keys to export value and length/size.                                                                    |            | no       |
 | `check_streams`               | `list(string)` | List of stream-patterns to export info about streams, groups, and consumers to search for with SCAN.                    |            | no       |
 | `check_single_streams`        | `list(string)` | List of single streams to export info about streams, groups, and consumers.                                             |            | no       |
+| `disable_exporting_key_values`| `bool`         | Whether to disable using values of keys stored in Redis as labels when using `check_keys` or `check_single_keys`.       |            | no       |
```

Comment:
```suggestion
| `disable_exporting_key_values`| `bool`         | Whether to disable using key values stored in Redis as labels when using `check_keys` or `check_single_keys`.       |            | no       |
```

Does this have a default? `false`? Or no default because it's not required?

Would it be better to approach it from the positive side and have it be:

`export_key_values` and default to `true`? And set the bool to `false` when you want to disable using stored key values?

Reply:
It was originally to match the upstream exporter, but updated to be `export_key_values`!

Reply:
Ah OK. Then whichever way makes sense for the implementation :-)